contracts/0.8.25/vaults/OperatorGrid.sol

```diff
@@ -372,7 +372,8 @@ contract OperatorGrid is AccessControlEnumerableUpgradeable, Confirmable2Address
     /// @param _requestedTierId id of the tier
     /// @param _requestedShareLimit share limit to set
     /// @return bool Whether the tier change was confirmed.
```

Comment:
```suggestion
    /// @return bool Whether the tier change was executed.
``` here and in other methods
Maybe worth explaining confirmations in more IQ-50 fashion.